### PR TITLE
Release 1.0.0-rc.0

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+## 1.0.0-rc.0 / 2016-07-14
+
+* [CHANGE] Remove deprecated query language keywords
+* [CHANGE] Change Kubernetes SD to require specifying Kubernetes role
+* [CHANGE] Use service address in Consul SD if available
+* [CHANGE] Standardize all Prometheus internal metrics to second units
+* [CHANGE] Remove unversioned legacy HTTP API
+* [CHANGE] Make PromQL vector matching to be ignoring by default
```
Comment (brian-brazil) [inline CHANGELOG.md]: This is an internal noop change, no need to mention it in the changelog. The `on()` is the feature it enables.

**Comment by brian-brazil** [comment]
:+1:

**Comment by beorn7** [comment]
Just for future reference: I suggest to follow the same tagging scheme as Go itself:
- bugfix releases have no RCs
- thus, this would be 1.0rc1
- 1.0.1 will be released without rc
- the rc for 1.1.0 would be called 1.1rc1

Cf. https://golang.org/dl/

**Comment by fabxc** [comment]
We follow semver tagging, which is well-defined and the scheme applied here.
http://semver.org/

**Comment by beorn7** [comment]
I see... then I have to blame the Go project to break the consistency. Too bad...

**Comment by fabxc** [comment]
I suppose that's, as many things, a derivative of some GOOG internals that shouldn't have made it into the outside world.

**Comment by beorn7** [comment]
I don't think so. I guess they wanted to do semver, but missed that point. So much the better that you did it right and fixed my faulty view of the world.

**Pull request merged**